_translator-files/po/sv-SE/Getting-Started.po

```diff
@@ -1,91 +1,92 @@
 # SOME DESCRIPTIVE TITLE
 # Copyright (C) YEAR Free Software Foundation, Inc.
 # This file is distributed under the same license as the PACKAGE package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
-#
-#, fuzzy
+# Daniel <daniel@mailsystem.net>, 2024.
+# ignotus <ignotus666@users.noreply.hosted.weblate.org>, 2024.
 msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"PO-Revision-Date: 2024-08-05 21:09+0000\n"
+"Last-Translator: ignotus <ignotus666@users.noreply.hosted.weblate.org>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
+"Language: sv-SE\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"X-Generator: Weblate 5.7-dev\n"
 
 #. type: Yaml Front Matter Hash Value: lang
 #: ../wiki/en/Getting-Started.md:1
 #, no-wrap
 msgid "en"
-msgstr ""
+msgstr "sv-SE"
 
 #. type: Yaml Front Matter Hash Value: layout
 #: ../wiki/en/Getting-Started.md:1
 #, no-wrap
 msgid "wiki"
-msgstr ""
+msgstr "wiki"
 
 #. type: Yaml Front Matter Hash Value: permalink
 #: ../wiki/en/Getting-Started.md:1
 #, no-wrap
 msgid "/wiki/Getting-Started"
-msgstr ""
+msgstr "/wiki/Getting-Started"
 
 #. type: Yaml Front Matter Hash Value: title
 #: ../wiki/en/Getting-Started.md:1
 #, no-wrap
 msgid "Setup"
-msgstr ""
+msgstr "Installation"
 
 #. type: Title #
 #: ../wiki/en/Getting-Started.md:8
 #, markdown-text, no-wrap
 msgid "Setup - getting started with Jamulus"
-msgstr ""
+msgstr "Installation - komma igång med Jamulus"
 
 #. type: Plain text
 #: ../wiki/en/Getting-Started.md:11
 #, markdown-text
 msgid "To get the best from Jamulus, at a minimum you will need:"
-msgstr ""
+msgstr "För att få ut det bästa av Jamulus behöver du minst:"
 
 #. type: Bullet: '1. '
 #: ../wiki/en/Getting-Started.md:15
 #, markdown-text
 msgid "**A wired internet connection** (and turn wi-fi off. See the [FAQ](/wiki/FAQ#why-shouldnt-i-use-wireless-equipment))"
-msgstr ""
+msgstr "**En trådbunden internetanslutning** (och stäng av Wi-Fi. Se [FAQ](/wiki/FAQ#why-shouldnt-i-use-wireless-equipment))"
 
 #. type: Bullet: '1. '
 #: ../wiki/en/Getting-Started.md:15
 #, markdown-text
 msgid "**Wired headphones** (not Bluetooth or speakers - see the [FAQ](/wiki/FAQ#why-shouldnt-i-use-wireless-equipment))"
-msgstr ""
+msgstr "**Trådbundna hörlurar** (inte Bluetooth eller högtalare – se [FAQ](/wiki/FAQ#why-shouldnt-i-use-wireless-equipment))"
```

Comment:
Translate text after # in link.

Reply:
`varför-bör-jag-inte-använda-trådlös-utrustning`